src/app/pages/offline-queue/offline-queue.page.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from "@angular/core";
```

Comment:
looks like all these double quotes actually need to be single quotes for the linter tests 👍